### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Rusik-best86**
- [x] Header height is set in 1 place (for the links)
- [x] Content is vertically centered (for any header height)
- [x] CSS is used to show all letters in Uppercase (don't type them in HTML)
- [x] Logo is an image wrapped with a link
- [x] CSS Variable is used for a blue color
- [x] Pseudo-element is used for a blue line below the active link
- [x] Code follows all the [Code Style Rules ❗️](https://github.com/Rusik-best86/layout_moyo-header/blob/master/checklist.md)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -5,10 +5,35 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__container">
+        <a class="header__logo" href="#"> <img src="/src/images/logo.png" alt="logo"></a>
```
Comment (BudnikOleksii) [inline src/index.html:17]: [Checklist](https://github.com/Rusik-best86/layout_moyo-header/blob/develop/checklist.md):
- alt attribute should describe the image content
image
- And fix formatting, don't put nested elements in the parent line

File: src/index.html [hunk 2]
```diff
@@ -5,10 +5,35 @@
...
+        <nav class="header__nav nav">
+          <ul class="nav__list">
+            <li class="nav__item"> <a class="nav__link is-active" href="#apple">Apple</a> </li>
+            <li class="nav__item"> <a class="nav__link"  href="#samsung">Samsung</a> </li>
+            <li class="nav__item"> <a class="nav__link" href="#smartphones">Smartphones</a> </li>
```
Comment (BudnikOleksii) [inline src/index.html:22]: - Add empty lines between multiline sibling blocks of HTML
image

File: src/index.html [hunk 3]
```diff
@@ -5,10 +5,35 @@
...
+            <li class="nav__item"> <a class="nav__link" href="#photo">Photo</a> </li>
+            <li class="nav__item"> <a class="nav__link" href="#video">Video</a> </li>
+          </ul>
+
+        </nav>
+      </div>
+
+    </header>
+
   </body>
```
Comment (BudnikOleksii) [inline src/index.html:37]: - But don't add them between parent and child elements.
image

**Review by BudnikOleksii — CHANGES_REQUESTED** [review]
Good work 👍 
Check the [checklist](https://github.com/Rusik-best86/layout_moyo-header/blob/develop/checklist.md) and fix your code accordingly.
And add links to the demo and test report in the PR:
image

**Comment by Rusik-best86** [comment]
kikikik

**Comment by Rusik-best86** [comment]
ikikik

**Comment by Rusik-best86** [comment]
9l9l9l

**Comment by Rusik-best86** [comment]
l9l

**Pull request closed without merging**